FinSearchUnified/BusinessLogic/Models/FindologicArticleModel.php

```diff
@@ -530,15 +530,19 @@ protected function setAttributes()
 
             $catPath = $this->seoRouter->sCategoryPath($category->getId());
 
+            foreach ($catPath as $key => $item) {
+                $catPath[$key] = str_replace('/', '', $item);
+            }
+
             while (!empty($catPath)) {
                 $tempPath = sprintf($routerCategoryTemplate, implode('/', $catPath));
 
-                if (Shopware()->Config()->get('routerToLower')) {
+                if (Shopware()->Config()->offsetGet('routerToLower')) {
                     $tempPath = mb_strtolower($tempPath);
                 }
 
                 if (!StaticHelper::isEmpty($tempPath)) {
-                    $categoryPath = $this->seoRouter->sCleanupPath($tempPath);
+                    $categoryPath = '/' . ltrim($this->seoRouter->sCleanupPath($tempPath), '/');
```

Comment:
Please add a comment above, why this is necessary, so this doesn't get removed in the future because someone thinks this is not necessary. I am also unsure about the last `/`, I may be wrong but I think there is a configuration, which allows  to change this.